slsDetectorSoftware/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ set(SOURCES
     src/CmdProxy.cpp
     src/CmdParser.cpp
     src/Pattern.cpp
+    ../slsSupportLib/opensslMd5/md5.c
```

Comment:
I think we should compile md5 as a part of the supportLib and then link it, instead of adding the sources here.